src/createServer.js

```diff
@@ -1,3 +1,28 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { parseUrlQuery } = require('./parseUrlQuery');
+const { errorCheck } = require('./errorCheck');
+const { createResponse } = require('./createResponse');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const urlQuery = req.url.split('?');
+
+    const { text, toCase } = parseUrlQuery(urlQuery);
+
+    const errorsMessages = errorCheck(text, toCase);
+    const isError = errorsMessages.errors.length !== 0;
+
+    res.setHeader('Content-Type', 'application/json');
+
+    res.statusCode = isError ? 400 : 200;
+    res.statusMessage = isError ? 'Bad request' : 'OK';
+
+    res.end(isError
+      ? JSON.stringify(errorsMessages)
+      : createResponse(text, toCase));
+  });
```

Comment:
You did this check 3 times)

```suggestion
    if (isError) {
       res.statusCode = 400;
       res.statusMessage = Bad request;
       res.end(JSON.stringify(errorsMessages));
       
       return;
    }
    
    res.statusCode = 200;
    res.statusMessage = 'OK';

    res.end(createResponse(text, toCase));
  });
```